src/main/java/Util.java

```diff
@@ -37,4 +37,27 @@
         return semesters;
     }
 
+    static String grabSequenceIDFromRequest(HttpServletRequest request) throws IOException {
```

Comment:
Don't mean to be pedantic but could you change this to `grabSequenceIdFromRequest`.  I think the lowercase `d` is easier to read.